system/Database/BaseConnection.php

```diff
@@ -1575,9 +1589,11 @@ abstract public function insertID();
     /**
      * Generates the SQL for listing tables in a platform-dependent manner.
      *
+     * @param false|string $tableName
```

Comment:
Please add explanation. What do you mean by table name? and its value false?

Reply:
Means default value is false, else define a string.

Reply:
I know these from the function signature.

`_listTables()` generates the SQL for listing tables.
It is not clear why tableName is needed to list tables.

Please update the doc comment.

Reply:
Please also don't mix types. If you really need a "no input" case it should be `null` instead of `false`name the parameter should be strongly typed as `?string = null`.

Reply:
Updated comment, change default to empty string to avoid type mismatch.